package.json

```diff
@@ -26,16 +28,20 @@
     "debug": "^3.1.0",
     "lodash": "^4.17.5",
     "request": "^2.87.0",
-    "request-promise": "^4.2.2"
+    "request-promise": "^4.2.2",
+    "ts-node": "^6.1.0"
```

Comment:
@et can we move this to devDependencies?

Reply:
Hi @pcothenet - sorry for the delay. I was on camping trip for the last week. I made that change for ya: https://github.com/brainflake/node-hubspot/pull/96